### DISCUSSION TITLE
how do i deploy a nuxt v3 project to the server？

### DISCUSSION THREAD

**Discussion by Abandoned9** [nuxt/nuxt#16344] [Questions]
I used the `build` command to generate the `.nuxt` folder, and there is also nodejs on the server, which files in the project should I put on the server?
How do i need to run it?

**Comment by shba007** [comment]
It will be far more simple if you can use following [hosting providers](https://nuxt.com/docs/getting-started/deployment#supported-hosting-providers) just upload your project to Github and go to the hosting providers and connect your Nuxt project from Github

If you want to manually deploy the content in .output folder and run node ./server/index.mjs from inside the .output folder for more details read this [Deployment Guidelines](https://nuxt.com/docs/getting-started/deployment)
